metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/templates/application.yml.j2

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+spring:
+  datasource:
+      driverClassName: org.h2.Driver
+      url: jdbc:h2:file:./metrondb
+      username: root
+      password: root
+      platform: h2
+  jpa:
+    hibernate:
+      ddl-auto: update
+
+zookeeper:
+  url:  {{zookeeper_quorum}}
+
+kafka:
+  broker:
+    url: {{kafka_brokers}}
+
+hdfs:
+  namenode:
+    url: {{default_fs}}
+
+grok:
+  path:
+    temp: ./patterns/temp
```

Comment:
That is not an HDFS path.  Think of it as a staging area when testing grok statements before they are persisted to HDFS.

That being said, I would consider our strategy for testing grok parsers to be out of scope for this PR.  Am happy to discuss is further but I think it should be in a different thread.